src/Service/Config.php

```diff
@@ -12,7 +16,8 @@ public function __construct() {
         if (file_exists(self::CONFIG_FILE)) {
             $this->config = json_decode(file_get_contents(self::CONFIG_FILE), true);
         } else {
-            throw new Exception("Config file not found: " . self::CONFIG_FILE);
+            //throw new Exception("Config file not found: " . self::CONFIG_FILE);
+            header("location: /setup.php");
```

Comment:
This exception should be added back and perhaps could have a custom one created `Tarkov\Exception\Config\NotFound`